nupic/research/frameworks/dendrites/modules/dendritic_mlp.py

```diff
@@ -37,12 +39,13 @@ class DendriticMLP(nn.Module):
 
     :param input_size: size of the input to the network
     :param output_size: the number of units in the output layer
-    :param hidden_size: the number of units in each of the two hidden layers
+    :param hidden_sizes: the number of units in each hidden layer
     :param num_segments: the number of dendritic segments that each hidden unit has
     :param dim_context: the size of the context input to the network
-    :param kw: whether to apply k-Winners (with 5% winners) to the outputs of each
-               hidden layer
-    :param dendrite_sparsity: the sparsity level of dendritic weights (default 0.95)
+    :param kw: whether to apply k-Winners to the outputs of each hidden layer
+    :param kw_percent_on: percent of hidden units activated by K-winners.
+    :param context_percent_on: percent of non-zero units in the context input.
+    :param weight_sparsity: the sparsity level of dendritic weights (default 0.95)
```

Comment:
A couple things:

1. the description for `weight_sparsity` seems outdated since you're also using this value to set the sparsity of feed-forward weights, and

2. I think the feedforward & dendritic weight sparsities should be kept separate, since I've often used different sparsity levels for these 2 sets of parameters.